### DISCUSSION TITLE
主界面显示全天时间进度条

### DISCUSSION THREAD

**Discussion by callingyou0** [ClassIsland/ClassIsland#563] [插件功能请求]
### 检查清单

- [X] 我已将本应用更新到最新测试版 ![最新的版本号](https://img.shields.io/github/v/release/ClassIsland/ClassIsland?include_prereleases&style=flat-square&label=)，并看过[最新提交](https://github.com/ClassIsland/ClassIsland/commits/dev/)，确认我想要的功能还没有实现。
- [X] 我已在 [Issues](https://github.com/ClassIsland/ClassIsland/issues?q=label%3A新功能) 和 [ClassIsland 文档](https://docs.classisland.tech/app/)中检索，确认我想要的功能未被提交过。
- [X] 我认为我想要的功能是用户广泛需要的，插件不能替代的，且没有超出 ClassIsland 作为**灵动课表**的开发目标，而非添加与课表及教学辅助无关的内容。（您可以学习[编写插件](https://docs.classisland.tech/dev/plugins/create-project/)来实现特殊的需求）。
- [X] 我已知晓由于开发者时间精力有限，加上已经积压了大量的功能请求，我提交的功能请求可能需要比较久的时间才能实现。

### 背景与动机

请看以下内容

### 想要实现或优化的功能

我希望可以增加一个功能. 

![哈哈](https://github.com/user-attachments/assets/f8cc9afa-67e4-4294-b75c-7b15d952b71d)
增加一个当前天的时间进度条或者是一个时间轴  可以更醒目的看当前处于一个什么时间节点并且下一个时间节点该干什么  比如说此图是我自己一天的规划  现在时间是晚上六点  加上这个进度条则可以马上看出来当前处于课间休息并且下一节课是计算机组成原理

此软件的开发特别有意义,不仅可以用到教学课堂内,而且考研人或者需要自学者也可当做时间规划来使用,希望开发人员可以往这方面多增加功能.

### 最后一步

- [X] 我认为上述的描述已经足以详细，以便开发人员理解您想要的功能。如果我的 issue 没有按照上述的要求填写，可能会被无条件关闭。

**Comment by DryIce-cc** [comment]
参见时间表中的时间线视图，（对于中小学生）这一横条可能会放不下…😂

> 考研人或者需要自学者也可当做时间规划来使用

这个需求超出了目前 ClassIsland 的开发目标，您可以学习[编写插件](https://docs.classisland.tech/dev/plugins/create-project)来实现此功能。

**Comment by FrankXiDong** [comment]
为什么还是待查看（？）
